.changeset/quiet-dolphins-double.md

```diff
@@ -0,0 +1,11 @@
+---
+'@leafygreen-ui/combobox': minor
+'@leafygreen-ui/date-picker': minor
+'@leafygreen-ui/number-input': minor
+'@leafygreen-ui/password-input': minor
+'@leafygreen-ui/select': minor
+'@leafygreen-ui/text-area': minor
+'@leafygreen-ui/text-input': minor
```

Comment:
I think just patch for these, no?

Reply:
yeah I guess these can all be patches since they're backwards-compatible